jsonrpc/dispatcher_test.go

```diff
@@ -25,6 +25,26 @@ func expectJSONResult(data []byte, v interface{}) error {
 	return nil
 }
 
+func TestDispatcherWebsocketStringId(t *testing.T) {
```

Comment:
This and `TestDispatcherWebsocketInvalidRequestFormats` are similar to each other and the targets are same. Can we put this 2 cases into one test by using subtests?

Reply:
Done